backend/src/controllers/squad.js

```diff
@@ -75,6 +77,12 @@ exports.postSquad = async (req, res, next) => {
 exports.putSquad = async (req, res, next) => {
   const updatedName = req.body.name;
 
+  if (!updatedName) {
+    res.status(400).send({
+      error: "No name provided."
+    });
+  }
+
```

Comment:
Bug I found during testing. If no name is provided it was returning a 404 which is incorrect.

Reply:
Good catch!